src/System.Private.Xml/src/System/Xml/Base64Encoder.cs

```diff
@@ -13,7 +13,7 @@ internal abstract partial class Base64Encoder
         private int _leftOverBytesCount;
         private char[] _charsLine;
 
-        internal const int Base64LineSize = 76;
+        internal const int Base64LineSize = 1024;
```

Comment:
> The default that's used is Base64FormattingOptions.None which means Convert isn't adding the new lines.

According to the [RFC](https://www.ietf.org/rfc/rfc2045.txt), not adding the new lines may be wrong.

> Encoded lines must not be longer than 76 characters,
          not counting the trailing CRLF. If longer lines are
          found in incoming, encoded data, a robust
          implementation might nevertheless decode the lines, and
          might report the erroneous encoding to the user.

As `76` is a meaningful number in RFC instead of a random number for buffer length, I don't think we can change it to 1024 for performance reason.